contrib/init/lnd.service

```diff
@@ -0,0 +1,50 @@
+# A sample systemd service file for lnd running with a bitcoind service.
+
+[Unit]
+Description=Lightning Network Daemon
+
+# Make sure lnd starts after bitcoind is ready
+Requires=bitcoind.service
+After=bitcoind.service
+
+[Service]
+ExecStart=/usr/local/bin/lnd
+ExecStop=/usr/local/bin/lncli stop
+PIDFile=/home/bitcoin/.lnd/lnd.pid
+
+# Replace these with the user:group that will run lnd
+User=bitcoin
+Group=bitcoin
+
+# Try restarting lnd if it stops due to a failure
+Restart=on-failure
+RestartSec=60
+
+# Type=notify is required for lnd to notify systemd when it is ready
+Type=notify
+
+# An extended timeout period is needed to allow for database compaction
+# and other time intensive operations during startup. We also extend the
+# stop timeout to ensure graceful shutdowns of lnd.
+TimeoutStartSec=1200
+TimeoutStopSec=3600
+
+# Hardening Measures
+####################
+
+# Mount /usr, /boot/ and /etc read-only for the process.
+ProtectSystem=full
```

Comment:
FYI, `ProtectHome=true` should also work as long as you set `ReadWriteDirectories=/home/bitcoin/.lnd/`